voila/handler.py

```diff
@@ -63,6 +65,17 @@ async def get(self, path=None):
         path, basename = os.path.split(notebook_path)
         notebook_name = os.path.splitext(basename)[0]
 
+        # Adding request uri to kernel env
+        self.kernel_env = os.environ.copy()
+        self.kernel_env['SCRIPT_NAME'] = self.request.path
+        self.kernel_env['PATH_INFO'] = ''  # would be /foo/bar if voila.ipynb/foo/bar was supported
+        self.kernel_env['QUERY_STRING'] = str(self.request.query)
+        self.kernel_env['SERVER_SOFTWARE'] = 'voila/{}'.format(__version__)
+        self.kernel_env['SERVER_PROTOCOL'] = str(self.request.version)
+        host, port = split_host_and_port(self.request.host.lower())
+        self.kernel_env['SERVER_PORT'] = str(port) if port else ''
+        self.kernel_env['SERVER_NAME'] = host
```

Comment:
Should these environment variables be namespaced (prefixed with `VOILA` for example).

Reply:
it follows https://tools.ietf.org/html/rfc3875

Reply:
ok then...